2996_reflection/reflection.md

```diff
@@ -4758,7 +4793,7 @@ consteval size_t size_of(info r);
 consteval size_t alignment_of(info r);
 ```
 
-[#]{.pnum} *Constant When*: `r` is a reflection representing an object, variable, type, non-static data member, or base class.
+[#]{.pnum} *Constant When*: `r` is a reflection representing an object, variable, type, non-static data member that is not a bit-field, or base class.
```

Comment:
Bitfields can have alignment, no?

Reply:
I don't believe so.

[[dcl.align]](https://eel.is/c++draft/dcl.align#1):
> An [alignment-specifier](https://eel.is/c++draft/dcl.attr.grammar#nt:alignment-specifier) may be applied to a variable or to a class data member, but it **shall not be applied to a bit-field**, a function parameter, or an [exception-declaration](https://eel.is/c++draft/except.pre#nt:exception-declaration) ([[except.handle]](https://eel.is/c++draft/except.handle))[.](https://eel.is/c++draft/dcl.align#1.sentence-1)

[Godbolt](https://godbolt.org/z/Ej3ETr7o6) suggests this is true as well:
```cpp
<source>:5:22: error: invalid application of 'alignof' to bit-field
    5 | static_assert(alignof(S::k) > 0);
```